tests/transports-integrations/tests/integrations/test_google.py

```
@@ -0,0 +1,472 @@
+10. Complete end2end test with conversation history, tool calls, tool results and images
+11. Integration specific tests
+"""
+
+import pytest
+import json
+import base64
+import requests
+from PIL import Image
+import io
+import google.generativeai as genai
+from typing import List, Dict, Any
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)